src/lib/layer-manager.js

```diff
@@ -142,19 +141,20 @@ export default class LayerManager {
     let redraw = false;
 
     // Make sure that buffer is cleared once when layer list becomes empty
-    if (this.layers.length === 0 && this.drewLayers) {
-      redraw = true;
-      return true;
-    }
-
-    if (this.redrawNeeded) {
-      this.redrawNeeded = false;
-      redraw = true;
+    if (this.layers.length === 0) {
+      if (this.layerCleared === false) {
+        redraw = true;
+        this.layerCleared = true;
+        return true;
+      }
+    } else {
+      if (this.layerCleared === true) {
```

Comment:
yes. you are right... it's the same no matter this if statement is here or not. 

I just wanted to make this line reads like "switch flip"... basically "if layerCleared is true, then flip it to false".